### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by Marblog** [community/community#61211] [GitHub Education]
### Hi everyone,

I have applied for several educational discounts but have not received recognition
![微信截图_20230719105620](https://github.com/community/community/assets/22953256/16a302af-8e48-4ef7-a7a0-dcbeaeba05fa)

**Comment by samus-aran** [comment]
Hi @Marblog 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Account or billing issues](https://support.github.com/request) Support and they will explain the details of this message.
